graphistry/feature_utils.py

```diff
@@ -230,18 +292,19 @@ def features_without_target(
     :param y: target DataFrame
     :return: DataFrames of model and target
     """
+    _, _, cudf = lazy_import_has_dependancy_cu_cat()
     if y is None:
         return df
     remove_cols = []
     if y is None:
         pass
-    elif isinstance(y, pd.DataFrame):
+    elif isinstance(y, pd.DataFrame) or isinstance(y, cudf.DataFrame):
```

Comment:
great catch

Reply:
@dcolinmorgan `or (cudf is not None and isinstance(y, cudf.DataFrame)`

Reply:
maybe same problem elsewhere?

Reply:
wouldnt that be a redundant check? this needs to be flexible for pd or cudf

Reply:
If cudf is None, I think the current would throw an exn